akka-testkit/src/main/scala/akka/testkit/TestActors.scala

```diff
@@ -19,6 +19,18 @@ object TestActors {
     }
   }
 
+  /**
+   * ForwardActor forwards all messages as-is to specified ActorRef.
+   *
+   * @param ref target ActorRef to forward messages to
+   */
+  class ForwardActor(ref: ActorRef) extends Actor {
+    override def receive = {
+      case message ⇒ ref forward message
+    }
+  }
+
   val echoActorProps = Props[EchoActor]()
+  val forwardActorProps: (ActorRef ⇒ Props) = Props(classOf[ForwardActor], _)
```

Comment:
I made the change.

By "evaluated once", I meant the evaluation that generates Function1, not the Props; i.e. there is a single Function1 with val, but a new Function1 every time an equivalent def is called. Somehow the former was appealing to me, to a healthy person it shouldn't make any difference :)